src/drf_yasg/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js

```diff
@@ -94,5 +94,5 @@ var r=n(301),o=n(512),i=n(512);t.applyOperation=i.applyOperation,t.applyPatch=i.
a function",r=NaN,o="[object Symbol]",i=/^\s+|\s+$/g,a=/^[-+]0x[0-9a-f]+$/i,s=/^0b[01]+$/i,u=/^0o[0-7]+$/i,c=parseInt,l="object"==typeof t&&t&&t.Object===Object&&t,f="object"==typeof self&&self&&self.Object===Object&&self,p=l||f||Function("return this")(),h=Object.prototype.toString,d=Math.max,m=Math.min,v=function(){return p.Date.now()};function g(e){var t=typeof e;return!!e&&("object"==t||"function"==t)}function y(e){if("number"==typeof e)return e;if(function(e){return"symbol"==typeof e||function(e){return!!e&&"object"==typeof e}(e)&&h.call(e)==o}(e))return r;if(g(e)){var t="function"==typeof e.valueOf?e.valueOf():e;e=g(t)?t+"":t}if("string"!=typeof e)return 0===e?e:+e;e=e.replace(i,"");var n=s.test(e);return n||u.test(e)?c(e.slice(2),n?2:8):a.test(e)?r:+e}e.exports=function(e,t,r){var o,i,a,s,u,c,l=0,f=!1,p=!1,h=!0;if("function"!=typeof e)throw new TypeError(n);function b(t){var n=o,r=i;return o=i=void 0,l=t,s=e.apply(r,n)}function _(e){var n=e-c;return void 0===c||n>=t||n<0||p&&e-l>=a}function w(){var e=v();if(_(e))return x(e);u=setTimeout(w,function(e){var n=t-(e-c);return p?m(n,a-(e-l)):n}(e))}function x(e){return u=void 0,h&&o?b(e):(o=i=void 0,s)}function E(){var e=v(),n=_(e);if(o=arguments,i=this,c=e,n){if(void 0===u)return function(e){return l=e,u=setTimeout(w,t),f?b(e):s}(c);if(p)return u=setTimeout(w,t),b(c)}return void 0===u&&(u=setTimeout(w,t)),s}return t=y(t)||0,g(r)&&(f=!!r.leading,a=(p="maxWait"in r)?d(y(r.maxWait)||0,t):a,h="trailing"in r?!!r.trailing:h),E.cancel=function(){void 0!==u&&clearTimeout(u),l=0,o=c=i=u=void 0},E.flush=function(){return void 0===u?s:x(v())},E}}).call(this,n(43))},function(e,t,n){var r={"./all.js":363,"./auth/actions.js":75,"./auth/index.js":325,"./auth/reducers.js":326,"./auth/selectors.js":327,"./auth/spec-wrap-actions.js":328,"./configs/actions.js":129,"./configs/helpers.js":162,"./configs/index.js":364,"./configs/reducers.js":333,"./configs/selectors.js":332,"./configs/spec-actions.js":331,"./deep-linking/helpers.js":164,"./deep-linking/index.js":334,"./deep-linking/layout.js":335,"./deep-linking/operation-tag-wrapper.jsx":337,"./deep-linking/operation-wrapper.jsx":336,"./download-url.js":330,"./err/actions.js":45,"./err/error-transformers/hook.js":104,"./err/error-transformers/transformers/not-of-type.js":309,"./err/error-transformers/transformers/parameter-oneof.js":310,"./err/index.js":307,"./err/reducers.js":308,"./err/selectors.js":311,"./filter/index.js":338,"./filter/opsFilter.js":339,"./layout/actions.js":85,"./layout/index.js":312,"./layout/reducers.js":313,"./layout/selectors.js":314,"./logs/index.js":322,"./oas3/actions.js":64,"./oas3/auth-extensions/wrap-selectors.js":343,"./oas3/components/callbacks.jsx":346,"./oas3/components/http-auth.jsx":352,"./oas3/components/index.js":345,"./oas3/components/operation-link.jsx":348,"./oas3/components/operation-servers.jsx":353,"./oas3/components/request-body-editor.jsx":351,"./oas3/components/request-body.jsx":347,"./oas3/components/servers-containe
```

Comment:
@hassansheikh What the change? This is not clear at all. It's very bad practice to hack compiled files.